src/openms/include/OpenMS/FORMAT/PepXMLFile.h

```diff
@@ -149,6 +149,12 @@ namespace OpenMS
     */
     void matchModification_(const double mass, const String& origin, String& modification_description);
 
+    /**
+        @brief create a TPP compatible string of the modified sequence using bracket notation
+        i.e.: n[35]RQLNK[162]LQHK[162]GEA
+    */  
+    static String modifiedSequenceToBracketString_(const AASequence & seq);
```

Comment:
pepXML is not the only place we have to read these square notation, another format is when reading SpectraST format. Also other search engines use it, for example ProteinPilot even though they have yet another notation. Basically the whole TPP uses it, its not just pepXML and we have to support it at the moment whether we like it or not.